chrome/content/zotero/xpcom/zotero.js

```diff
@@ -381,7 +381,26 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 					return false;
 				}
 			}
-			
+			if(Zotero.File.isDropboxDirectory(dataDir.path) && Zotero.Prefs.get('warnDropboxDataDir')){
+				let check = {value: false};
+				ps = Services.prompt;
```

Comment:
I guess the use of `ps` below, though given the brevity of `Services.prompt`, this seems unnecessary